src/components/organisms/PageHeader/PageHeader.tsx

```diff
@@ -218,20 +259,29 @@ const PageHeader = () => {
   const helmValuesMap = useAppSelector(state => state.main.helmValuesMap);
   const helmChartMap = useAppSelector(state => state.main.helmChartMap);
   const previewType = useAppSelector(state => state.main.previewType);
-  const isKubeconfigPathValid = useAppSelector(state => state.config.isKubeconfigPathValid);
-  const kubeConfig = useAppSelector(state => state.config.kubeConfig);
-  const clusterStatusHidden = useAppSelector(state => state.ui.clusterStatusHidden);
   const previewLoader = useAppSelector(state => state.main.previewLoader);
-  const kubeconfigPath = useAppSelector(state => state.config.kubeconfigPath);
+  const projects: Project[] = useAppSelector(state => state.config.projects);
 
   const isInPreviewMode = useSelector(isInPreviewModeSelector);
   const isInClusterMode = useSelector(isInClusterModeSelector);
-  const isClusterActionDisabled = !kubeconfigPath || !isKubeconfigPathValid;
 
   const [previewResource, setPreviewResource] = useState<K8sResource>();
   const [previewValuesFile, setPreviewValuesFile] = useState<HelmValuesFile>();
   const [helmChart, setHelmChart] = useState<HelmChart>();
   const dispatch = useAppDispatch();
+  const activeProject = useSelector(activeProjectSelector);
+  const highlightedItems = useAppSelector(state => state.ui.highlightedItems);
+  const currentConfig = useAppSelector(currentConfigSelector);
+  const projectConfig = useAppSelector(state => state.config.projectConfig);
```

Comment:
what's the difference between `currentConfig` and `projectConfig` ?

Reply:
I will explain with a comment at the bottom